src/constants_and_utils/utils.js

```diff
@@ -17,8 +19,46 @@ function waitForElementToBeClickable(htmlElement, timeoutInMilliseconds, errorMe
   browser.wait(EC.elementToBeClickable(htmlElement), timeoutInMilliseconds, errorMessage);
 }
 
+function protractorHelperWarning() {
+  return colors.yellow("Protractor-helper warning: ");
+}
+
+function readMoreDoc() {
+  return colors.cyan(" [Read more on www.npmjs.com/package/protractor-helper#preparation-to-next-major-version]");
+}
+
+function obsoleteFunction(functionName, text = "") {
+  console.warn(
+    protractorHelperWarning() +
+      "Function '" +
+      functionName +
+      "' will be deprecated in version 4.0.0! " +
+      text +
+      readMoreDoc()
+  );
+}
+
+function replaceObsoleteFunction(oldFunctionName, newFunctionName) {
+  obsoleteFunction(oldFunctionName, "Please use the new '" + newFunctionName + "' function instead!");
```

Comment:
What about using ES6 syntax for string concatenation?

I recommend reading https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/template_strings.

In the [messageBuilder](https://github.com/wlsf82/protractor-helper/blob/master/src/constants_and_utils/messageBuilder.js) you can find some examples.

Reply:
I will do it.